File: SERVER/foody-admin/src/admin/modules/user/kind/Kind.php
<?php

class Kind
{
    private $kindID;
    private $kindCode;
    private $kindName;
    private $create_at;
    private $update_at;

    // public function __construct()
    // {
    // }

    // public function __construct($kindID, $kindCode, $kindName)
    // {
    //     setKindID($kindID);
    //     setKindCode($kindCode);
    //     setKindName($kindName);
    // }

    /**
     * Get the value of kindName.
     */
    public function getKindName()
    {
        return $this->kindName;
    }

    /**
     * Set the value of kindName.
     *
     * @return self
     */
    public function setKindName($kindName)
    {
        $this->kindName = $kindName;

        return $this;
    }

    /**
     * Get the value of kindCode.
     */
    public function getKindCode()
    {
        return $this->kindCode;
    }

    /**
     * Set the value of kindCode.
     *
     * @return self
     */
    public function setKindCode($kindCode)
    {
        $this->kindCode = $kindCode;

        return $this;
    }

    /**
     * Get the value of kindID.
     */
    public function getKindID()
    {
        return $this->kindID;
    }

    /**
     * Set the value of kindID.
     *
     * @return self
     */
    public function setKindID($kindID)
    {
        $this->kindID = $kindID;

        return $this;
    }

    /**
     * Get the value of create_at.
     */
    public function getCreate_at()
    {
        return $this->create_at;
    }

    /**
     * Set the value of create_at.
     *
     * @return self
     */
    public function setCreate_at($create_at)
    {
        $this->create_at = $create_at;

        return $this;
    }

    /**
     * Get the value of update_at.
     */
    public function getUpdate_at()
    {
        return $this->update_at;
    }

    /**
     * Set the value of update_at.
     *
     * @return self
     */
    public function setUpdate_at($update_at)
    {
        $this->update_at = $update_at;

        return $this;
    }
}

File: APP/Foody/app/src/main/java/com/gmail/khanhit100896/foody/kind/Kind.java
package com.gmail.khanhit100896.foody.kind;

public class Kind {

    private int id;
    private String kind_code;
    private String kind_name;

    public Kind(int id, String kind_code, String kind_name) {
        this.setId(id);
        this.setKind_code(kind_code);
        this.setKind_name(kind_name);
    }

    public int getId() {
        return id;
    }

    private void setId(int id) {
        this.id = id;
    }

    public String getKind_code() {
        return kind_code;
    }

    private void setKind_code(String kind_code) {
        this.kind_code = kind_code;
    }

    public String getKind_name() {
        return kind_name;
    }

    private void setKind_name(String kind_name) {
        this.kind_name = kind_name;
    }
}
